tests/phpunit/tests/option/transient.php

```diff
@@ -184,4 +184,62 @@ public function test_nonexistent_key_old_timeout() {
 		);
 		$this->assertSame( $expected, $a->get_events() );
 	}
+
+
+	/**
+	 * @dataProvider data_transient_keys
+	 *
+	 * @covers ::set_transient()
+	 */
+	public function test_transient_key_valid( $key, $expected, $message ) {
+		$value  = rand_str();
+
+		$this->assertSame( $expected, set_transient( $key, $value ), $message );
+	}
+
+	public function data_transient_keys() {
```

Comment:
We need a docblock with a ticket reference here :)

Reply:
apologies on this one, as I looked up examples of `data_*` functions in the tests and saw they were often not doc-bloc'ed. I assumed wrongly that you only doc-bloc the function that uses the data. fixed